server/api/utils/clients/iguazio.py

```diff
@@ -755,17 +755,18 @@ def _transform_mlrun_project_to_iguazio_project(
             # This feature requires support for project-level default_function_node_selector,
             # which is available starting from version 3.5.5.
             # We are adding this validation to maintain backward compatibility with older versions of Iguazio.
-            if mlrun.utils.helpers.is_igz_version_sufficient("3.5.5"):
+            if mlrun.utils.helpers.validate_component_version_compatibility("3.5.5"):
```

Comment:
I think you missed the 1st param
```suggestion
            if mlrun.utils.helpers.validate_component_version_compatibility("iguazio", "3.5.5"):
```